tests/Microsoft.DotNet.Framework.Docker.Tests/AspnetImageTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class AspnetImageTests
+    {
+        private static ImageDescriptor[] AspnetTestData = new ImageDescriptor[]
+        {
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "3.5", OsVariant = OsVersion.WSC_1903 },
+            new ImageDescriptor { Version = "4.6.2", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.1", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_1903 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2016 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_1803 },
+            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2019 },
+            new ImageDescriptor { Version = "4.8",  OsVariant = OsVersion.WSC_1903 },
+        };
+
+        private readonly ImageTestHelper imageTestHelper;
+
+        public AspnetImageTests(ITestOutputHelper outputHelper)
+        {
+            imageTestHelper = new ImageTestHelper(outputHelper);
+        }
+
+        [Theory]
+        [Trait("Category", "aspnet")]
+        [MemberData(nameof(GetVerifyAspnetImagesData))]
+        public void VerifyAspnetImagesWithApps(ImageDescriptor imageDescriptor)
```

Comment:
I like the generic `VerifyImagesWithApps` name used in the RuntimeSdkImageTests.